recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -174,8 +174,6 @@ this.RecipeRunner = {
         resolve(clonedResult);
       });
       sandboxManager.addGlobal("actionFailed", err => {
-        Cu.reportError(err);
```

Comment:
It does get reported, although I think it gets reported via the "you didn't handle a promise rejection and it happened, shame on you" mechanism. I'll followup with another PR to properly handle that rejection.